app/main/views/briefs.py

```diff
@@ -226,6 +226,10 @@ def redirect_to_next_page():
                 return redirect(url_for('.view_response_result', brief_id=brief_id))
 
     previous_question_id = section.get_previous_question_id(question_id)
+    # Skip previous question if the brief has no nice to have requirements
+    if previous_question_id in brief.keys() and not brief[previous_question_id]:
```

Comment:
I'm not sure I understand this condition. Why would `nicToHaveRequirements` be in the brief.keys if brief doesn't have any? Could we check for `if not brief.get(previous_question_id):` instead?